src/ray/raylet/worker_pool_test.cc

```diff
@@ -61,6 +87,12 @@ class WorkerPoolTest : public ::testing::Test {
     return std::shared_ptr<Worker>(new Worker(pid, language, client));
   }
 
+  void SetWorkerCommands(
+      const std::unordered_map<Language, std::vector<std::string>> &worker_commands) {
+    WorkerPoolMock worker_pool(worker_commands);
+    this->worker_pool_ = std::move(worker_pool);
+  }
```

Comment:
just pass worker_commands to the constructor?

Reply:
But I couldn't find a way to pass the arguments into the object of a fixture mock class.
I think it's ok for a test.